lib/foods.js

```diff
@@ -46,7 +46,18 @@ function clearErrors(){
 }
 
 function addDeleteFunctionality(){
-  $("td.food-delete button").on('click', function(){
+  $("#food-list").on('click', "button", function(){
+    var row = $(this).parentsUntil("tbody")
+    var item = row[1].firstElementChild.innerHTML
+    var foodItems = JSON.parse(localStorage.getItem("food-items"))
```

Comment:
@case-eee I had tried pulling this out into it's own function and a few things below this as well and found that something got really screwed up with the localStorage. Will revisit! thanks!